parallel-consumer-core/src/test-integration/java/io/confluent/parallelconsumer/integrationTests/state/PartitionStateCommittedOffsetIT.java

```diff
@@ -69,6 +72,8 @@ class PartitionStateCommittedOffsetIT extends BrokerIntegrationTest<String, Stri
 
     private OffsetResetStrategy offsetResetStrategy = DEFAULT_OFFSET_RESET_POLICY;
 
+    private ParallelEoSStreamProcessor<String, String> activePc;
```

Comment:
consider returning PC instance from runPcCheckStartIs method instead of using handover through that private var.
It's only used in one test - to check failure cause - on offset reset policy NONE.

Reply:
wrong method referenced in comment - correct one is runPcUntilOffset - which already has a return type. So it really up to you to either wrap returned value (as Pair /Tuple / custom class) to hold both existing return value and in addition to it the ParallelEoSStreamProcessor or just leave as is...